api/Window.json

```diff
@@ -1694,54 +1694,6 @@
           }
         }
       },
-      "dialogArguments": {
```

Comment:
IE, Safari and Firefox had `window.dialogArguments`

Reply:
_If_ the removal of `showModalDialog` was accompanied by the removal of `dialogArguments`, yes probably.
But like I said Firefox supported `dialogArguments` hence the table would be erroneous.
Someone with access to a browser farm will have to test it out.